docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
         #WHITENOISE_KEEP_ONLY_HASHED_FILES: 'True'
 
     # WhiteNoise needs --nostatic for development
-    command: 'django-admin runserver --nostatic 0.0.0.0:8000'
```

Comment:
WhiteNoise requires `--nostatic` to work with `runserver`:

http://whitenoise.evans.io/en/stable/django.html#using-whitenoise-in-development